cypress.config.js

```diff
@@ -36,31 +40,80 @@ const getDevBaseUrl = () => {
       'To test a dev URL, add the `baseUrl` property to your `DEV` portal configuration in `hubspot.config.yml`',
     );
   }
-
   return null;
 };
 
-const config = {
-  env: {
-    env: currentEnv,
-    isProd: currentEnv === PROD,
-    isQA: currentEnv === QA,
-    isDev: currentEnv === DEV,
+
+/**
+ * @returns {object} The configuration object for handling webpack extensions
+ */
+const setupNodeEvents = async (on, config) => {
```

Comment:
Not sure why is not picking this up 

cc @darkmavis1980 any idea? it is referenced bellow but still references the function coming from cypress

Reply:
Because it's not using it, if you look at [this line](https://github.com/HubSpotWebTeam/wt-eslint-browser/pull/11/files#diff-5bae461d15457335f3d6285d35d3df2fd618e583eb7460c52cd8f0de69627abcR83), it's not using this function, but it's just defining a method with the same name, not reusing this one ;-)

Reply:
Okay, took me a minute to figure out what was going on here, but looking at [these docs](https://github.com/Shelex/cypress-allure-plugin/blob/master/README.md) seemed to clear it up.

I believe the intention was to replace the function, and run the preprocessor stuff with webpack. However, we can set the webpack options outside of the `e2e` object, and just call the `on('file:preprocessor, webpack(options))` with the default function. Only change I think is to run the setupNodeEvents Async, so we can await the cucumber plugins.

```js
const e2e = {
  specPattern: 'cypress/e2e/*.cy.js',
  testIsolation: false,
  async setupNodeEvents(on, config) {
    await addCucumberPreprocessorPlugin(on, config);
    on('file:preprocessor', webpack(webpackOptions));
    allureWriter(on, config);
    return config
  },
};
```